components/AddTierModal/index.js

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+
+const ModalContainer = styled.div`
+.absolute;
+.top-50%;
+.left-50%;
+transform: translate(-50%,0%);
+background: white;
+.rounded;
+width:40vw;
+height:400px;
```

Comment:
with fixed height, dont you think, we need an `overflow-y: scroll`?

Reply:
Yes, I will add that property.